src/backend/main.ts

```diff
@@ -1275,7 +1276,13 @@ ipcMain.handle(
     const { title } = gameManagerMap[runner].getGameInfo(appName)
     notify({ title, body: i18next.t('notify.moving', 'Moving Game') })
 
-    const moveRes = await gameManagerMap[runner].moveInstall(appName, path)
+    let moveRes: InstallResult = {
+      status: 'error',
+      error: 'No flatpak sandbox access to this path'
+    }
+
+    if (isAccessibleWithinFlatpakSandbox(path as Path))
```

Comment:
Doing `path as Path` means that `path` won't actually be verified to be a `Path`. Use `Path.parse(path)` instead (or perhaps `safeParse`)